packages/sdk/src/utils/constants.ts

```diff
@@ -26,3 +36,102 @@ export const NULL_BYTES32 = '0x0000000000000000000000000000000000000000000000000
+  MintRandomnessAlreadyRevealed,
+  NoAddressesToAirdrop,
+  MintHasConcluded,
+  MintsAlreadyExist,
+  Underpaid,
+  ExceedsAvailableSupply,
+  MintNotOpen,
+  MintPaused,
+  InvalidTimeRange,
+  Unauthorized,
+  InvalidAffiliateFeeBPS,
+  FeeRegistryIsZeroAddress,
+  InvalidMaxMintableRange,
+  ExceedsMaxPerAccount,
+  MaxMintablePerAccountIsZero,
+  InvalidMerkleProof,
+  MerkleRootHashIsEmpty,
+} = ContractErrorNames
+
+// Maps contract error signature hashes to error names
+export const ContractErrorSigNames = {
+  // SoundEdition
+  [editionInterface.getSighash(MetadataIsFrozen)]: MetadataIsFrozen,
+  [editionInterface.getSighash(InvalidRoyaltyBPS)]: InvalidRoyaltyBPS,
+  [editionInterface.getSighash(InvalidRandomnessLock)]: InvalidRandomnessLock,
+  [editionInterface.getSighash(ExceedsEditionAvailableSupply)]: ExceedsEditionAvailableSupply,
+  [editionInterface.getSighash(InvalidAmount)]: InvalidAmount,
+  [editionInterface.getSighash(InvalidFundingRecipient)]: InvalidFundingRecipient,
+  [editionInterface.getSighash(InvalidEditionMaxMintableRange)]: InvalidEditionMaxMintableRange,
+  [editionInterface.getSighash(MaximumHasAlreadyBeenReached)]: MaximumHasAlreadyBeenReached,
+  [editionInterface.getSighash(ExceedsAddressBatchMintLimit)]: ExceedsAddressBatchMintLimit,
+  [editionInterface.getSighash(MintRandomnessAlreadyRevealed)]: MintRandomnessAlreadyRevealed,
+  [editionInterface.getSighash(NoAddressesToAirdrop)]: NoAddressesToAirdrop,
+  [editionInterface.getSighash(MintHasConcluded)]: MintHasConcluded,
+  [editionInterface.getSighash(MintsAlreadyExist)]: MintsAlreadyExist,
+  // IMinterModule
+  [iMinterModuleInterface.getSighash(Underpaid)]: Underpaid,
+  [iMinterModuleInterface.getSighash(ExceedsAvailableSupply)]: ExceedsAvailableSupply,
+  [iMinterModuleInterface.getSighash(MintNotOpen)]: MintNotOpen,
+  [iMinterModuleInterface.getSighash(MintPaused)]: MintPaused,
+  [iMinterModuleInterface.getSighash(InvalidTimeRange)]: InvalidTimeRange,
+  [iMinterModuleInterface.getSighash(Unauthorized)]: Unauthorized,
+  [iMinterModuleInterface.getSighash(InvalidAffiliateFeeBPS)]: InvalidAffiliateFeeBPS,
+  [iMinterModuleInterface.getSighash(FeeRegistryIsZeroAddress)]: FeeRegistryIsZeroAddress,
+  // RangeEditionMinter
+  [rangeMinterInterface.getSighash(InvalidMaxMintableRange)]: InvalidMaxMintableRange,
+  [rangeMinterInterface.getSighash(ExceedsMaxPerAccount)]: ExceedsMaxPerAccount,
+  [rangeMinterInterface.getSighash(MaxMintablePerAccountIsZero)]: MaxMintablePerAccountIsZero,
+  // MerkleDropMinter
+  [merkleMinterInteface.getSighash(InvalidMerkleProof)]: InvalidMerkleProof,
+  [merkleMinterInteface.getSighash(ExceedsMaxPerAccount)]: ExceedsMaxPerAccount,
+  [merkleMinterInteface.getSighash(MerkleRootHashIsEmpty)]: MerkleRootHashIsEmpty,
+  [merkleMinterInteface.getSighash(MaxMintablePerAccountIsZero)]: MaxMintablePerAccountIsZero,
+} as const
+
+export type ContractErrorName = typeof ContractErrorSigNames[keyof typeof ContractErrorSigNames]
```

Comment:
```suggestion
export type ContractErrorSigName = typeof ContractErrorSigNames[keyof typeof ContractErrorSigNames]
```

Reply:
Hopefully this makes it more clear?: https://github.com/soundxyz/sdk/pull/175/commits/5c11866b0042dedffa2f67850b177b38dc5f1bbe

`ContractErrorSigHashToName` is a mapping from hashes to names, whereas `ContractErrorName` is just a union of names.

Reply:
> both enums have to be called differently (not only s suffix) and exported separetly

@PabloSzx I'm using the mapping as the source of truth since it is what gets used in `getContractError`, but they're the same type: z

![image](https://user-images.githubusercontent.com/32403632/210314270-a46405f2-2ed3-4957-ba87-04866dc396e4.png)
![image](https://user-images.githubusercontent.com/32403632/210314285-9893d2eb-414c-4a10-af78-7ed2bc2869b3.png)